examples/seismic/abc_methods/03_pml.ipynb

```diff
@@ -384,7 +384,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The velocity field is needed in both staggered and non-staggered grids. As before, we read the file and interpolate it to the non-staggered grid. From these values we interpolate to the staggered grid."
+    "The velocity field is needed in both staggered and non-staggered grids. We read the file and interpolate it to the non-staggered grid. From these values, we interpolate to the staggered grid."
```

Comment:
Maybe `before` can stay

Reply:
I took another look and I agree with you